src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -350,7 +350,7 @@ public string GetFileShareLocationForReleaseJson(DownloadedBuild build)
             { $"{githubRepoPrefix}dotnet/runtime", (coreRepoCategory, "runtime") },
             { $"{githubRepoPrefix}dotnet/windowsdesktop", (coreRepoCategory, "windowsdesktop") },
             { $"{githubRepoPrefix}dotnet/templating", (coreRepoCategory, "templating") },
-            { $"{githubRepoPrefix}dotnet/emsdk", (coreRepoCategory, "emsdk") },
```

Comment:
we are removing the github version of emsdk?

Reply:
Doy...no.